service/application/src/main/resources/application.yml

```diff
@@ -28,6 +28,10 @@ spring:
       enabled: true
       settings:
         web-allow-others: true
+  servlet:
+    multipart:
+      max-request-size: 5MB
+      max-file-size: 5MB
```

Comment:
Shouldn't we allow this to be configured? 5MB might be way too small for some scans.